server/database/config/build.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
+
+    DROP TABLE IF EXISTS user, event, news_tag, topic, event_topic, event_category, primary_tag, secondary_tag, news CASCADE;
+
+    CREATE TABLE user (id SERIAL PRIMARY KEY, first_name TEXT NOT NULL, last_name TEXT NOT NULL, email UNIQUE TEXT NOT NULL, password TEXT NOT NULL, avatar TEXT, business_type TEXT NOT NULL, website TEXT NOT NULL, organisation_name TEXT NOT NULL, address TEXT NOT NULL, city TEXT NOT NULL, country TEXT NOT NULL, postal_code TEXT NOT NULL, facebook TEXT NOT NULL, instagram TEXT NOT NULL, twitter TEXT NOT NULL);
+
+    CREATE TABLE event (id SERIAL PRIMARY KEY, title TEXT NOT NULL, description TEXT NOT NULL, category INTEGER NOT NULL REFERENCES event_category(id), event_datetime DATETIME NOT NULL, venue TEXT NOT NULL, website TEXT NOT NULL, cost TEXT NOT NULL, image TEXT NOT NULL, focus_key TEXT NOT NULL, meta TEXT NOT NULL, alt_text TEXT NOT NULL, is_draft BOOLEAN NOT NULL, publisher_id INTEGER NOT NULL REFERENCES user(id), publish_datetime DATETIME NOT NULL);
+
+    CREATE TABLE news_tag (secondary_tag INTEGER NOT NULL REFERENCES secondary_tag(id), news_id INTEGER NOT NULL REFERENCES news(id));
+
+    CREATE TABLE topic (id SERIAL PRIMARY KEY, topic TEXT NOT NULL);
+
+    CREATE TABLE event_topic (event_id INTEGER NOT NULL REFERENCES event(id), topic_id INTEGER NOT NULL REFERENCES topic(id));
```

Comment:
that cannot be null because it's reference from existence 
```suggestion
    CREATE TABLE event_topic (event_id INTEGER REFERENCES event(id), topic_id INTEGER NOT NULL REFERENCES topic(id));
```